### ISSUE TITLE
Line comments appear in editing mode in the programming-exercise assessment editor

### ISSUE THREAD

**Issue by moritzhertler** [ls1intum/Artemis#3455]
### Describe the bug
When switching between files, sometimes already written and saved comments randomly appear in editing mode, but the text box is empty and does not show the original content of the comment. All buttons are not responsive. Saving the work and reloading resolves the issue. 

#### To Reproduce

The issue appears randomly in different submissions. I could not find a pattern.

### Environment

<details><pre>
 - OS: Windows 10 20H2
 - Browser: Chrome 
 - Version: 90.0.4430.212 
</pre></details>

#### Additional context
I am a Tutor at KIT, @dfuchss is my supervisor.

#### Output in Web Console

<details><pre>
   None.
</pre></details>

**Comment by krusche** [comment]
Same as in the other issues. Screenshots or better screencasts would help

**Comment by moritzhertler** [comment]
I started a new assessment and added/edited random comments until the issue appeared again. 

**The screencast can be viewed [here](https://drive.google.com/file/d/1inC-CXfkwLG2MMZjQgwSDI2kUGLRmg2g/view?usp=sharing).** 

Short description of the screencast:

- I had already added a few comments before this recording. The cast starts with me editing existing comments that have already been saved.
- I switch between files and edit/add comments for 45 seconds.
- At 00:45 I open Month.java and the comment on line 11 which should contain "Test 13" is displayed in editing mode without any text in the text box. The cancel and save buttons do not work but the number and text inputs accept inputs. 
- After reloading at 01:10 (unfortunately without saving in beforehand) the comment is displayed normally again and contains its original content. All other comments are in their expected state (all unsaved changes are lost). In previous cases, saving worked as expected (all changes are saved and the affected comment's state does not change).

It is probably a coincidence but all affected comments so far have had a score of 0 points. This could be because I use comments with a 0 points score way more than comments with a negative or positive score. 

I do not know if this could be relevant but I usually have the assessment dashboard opened in a second tab to get an overview of how many exercises I have already graded.  As far as I remember, the issue only appeared when the second Artemis tab is opened (which is most of the time so likely also a coincidence).
